dfanalyzer/main.py

```diff
@@ -542,9 +542,19 @@ def _remove_numbers(self, string_items):
         logging.info(f"List after removing numbers {list(item_sets)}")
         return list(item_sets)
 
+    def _check_hosts_time_skew(self):
+        # check if there is time skew across nodes
+        hosts_ts_df = self.events.groupby('hostname').agg({'ts': 'min'}).compute()
+        # filter the hosts if time skew exceeds 30 seconds
+        max_time_skew = 30e6
+        if np.std(hosts_ts_df['ts']) > max_time_skew:
+           logging.warn(f"The time skew exceeds {max_time_skew // 1e6} sec across hosts {hosts_ts_df.index.tolist()}")
+
     def summary(self):
         num_events = len(self.events)
         logging.info(f"Total number of events in the workload are {num_events}")
+        if len(self.events["hostname"].unique().compute()) > 1:
+            self._check_hosts_time_skew()
```

Comment:
could u move this past 571 `hosts_used = hosts_used.to_list()`
```
       if len(hosts_used) > 1:
            self._check_hosts_time_skew()
```

Reply:
I submitted the changes, please use [6e020f8](https://github.com/hariharan-devarajan/dftracer/pull/163/commits/6e020f8af369dc819d4965a7f224803aeed69d0f)